### PR TITLE
Study case - Vite js as development platform

### PR DESCRIPTION
**Pull request by LAakerberg**
My study case is about Vite js as a development platform.

### CODE REVIEW

**Review by Fermain — CHANGES_REQUESTED** [review]
This is a highly detailed and well constructed case study on a crucial tool. There is only one issue highlighted below. This is a matter of tone rather than content. Overall I enjoyed reading your case study. Thank you for your contribution, we will be able to merge with this one change.

File: vitejs-case-study.md [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+3. Features
+4. Strengths
+5. Weaknesses
+6. Comparison
+7. Summary
+8. Reference
+
+## Introduction
+
+I have chosen to learn more about what a bundler is and why we need to have them. I wanted to dig in and learn more about the building tool Vite js.
```
[the remark this answers is not in the document]
Comment (LAakerberg) [reply vitejs-case-study.md]: Updated have been done with new introduction.

**Comment by LAakerberg** [comment]
Changes have been done.

**Review by Fermain — APPROVED** [review]
Thank you for this detailed and informative contribution

**Pull request merged**